workflows/reference/gatk-reference.wdl

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+import "../../tools/picard.wdl"
+import "../../tools/samtools.wdl"
+import "../../tools/util.wdl"
+
+workflow gatk_reference {
+    meta {
+        description: "Fetches reference files for GATK."
+        outputs: {
+            fasta: "FASTA file for the reference genome."
+            fasta_index: "Index for the FASTA file for the reference genome."
+            fasta_dict: "Picard Sequence Dictionary for the reference genome."
```

Comment:
Are `.dict`s a Picard specific format?

Reply:
Not anymore. I think it was originally a Picard file, but now you can generate one with `samtools` as well.